src/main/java/training/subsystems/Arm/Elbow/Elbow.java

```diff
@@ -0,0 +1,36 @@
+package training.subsystems.Arm.Elbow;
+
+import edu.wpi.first.math.geometry.Rotation2d;
+import utils.GBSubsystem;
+
+public class Elbow extends GBSubsystem {
+    protected static Elbow instance;
+    protected static IElbow iElbow;
```

Comment:
```suggestion
    private IElbow iElbow;
```
u dont want someone to access ur motor